example/cmd/device-simple/res/Simple-Driver.yaml

```diff
@@ -49,35 +52,18 @@ deviceResources:
 deviceCommands:
```

Comment:
You miss the `isHidden: false` in deviceCommands.
This is an example, so we have to demonstrate the sample clearly although the default value is false

Reply:
Added the isHidden fields.